django/asapsports/db/notifications.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+
+def insert_notification(conn, start_time, game_id):
+    print("start")
+    query = """ insert into notifications (start_time, game_id)
+                  values (%(start_time)s, %(game_id)s)  """
+    with conn.cursor() as curs:
+        curs.execute(query, locals())
+
+def get_current_notification(conn, cur_time):
+    query = """ select * from notifications"""
```

Comment:
select * isn't very readable

Reply:
dw about it ima rewrite it later

Reply:
`def get_current_notification(conn):
    query = """ SELECT game_id FROM notifications 
        WHERE start_time - INTERVAL '1 hour' >= current_timestamp """
    with conn.cursor() as curs:
        curs.execute(query)
        return [row[0] for row in curs]
`

Reply:
we might need to re write this again if we do more refactoring on the app